packages/@react-stately/tree/src/useTreeState.ts

```diff
@@ -26,13 +29,13 @@ export interface TreeState<T> {
   readonly disabledKeys: Set<Key>,
 
   /** A set of keys for items that are expanded. */
-  readonly expandedKeys: Set<Key>,
+  readonly expandedKeys: 'all' | Set<Key>,
```

Comment:
changing this type could potentially be a breaking change. if someone was relying on expandedKeys being a set (and doing things like `expandedKeys.has('foo')` on it), then the `'all'` value would cause an error to be thrown.

Reply:
ah, good call, thats a bit annoying since it would be good for this to be consistent with our expandable rows table api... I can get rid of this `'all'` support for now, but I guess I would have to create a separate tree state hook then to support `'all'`, thoughts?

Reply:
where did the requirement for `all` come from? We don't have UI for an "expand all" button right?

Reply:
we don't have a hard requirement for `all` but it was added to the expandable table rows to help with the eventual async case where a team may want all their rows to be auto expanded as they are loaded in